x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -133,7 +158,7 @@ export class TelemetryEventsSender {
     return this.isOptedIn === true;
   }
 
-  private async sendIfDue() {
+  public async sendIfDue(axiosInstance: AxiosInstance = axios) {
```

Comment:
NOTE: I do have to make this `public` but I don't think this is a big ordeal. Later it might be able to go back to `private` if we change the task runners to not do correlations/joins within the task runners.